app/src/main/java/com/postpc/tenq/services/RecorderService.java

```diff
@@ -0,0 +1,138 @@
+package com.postpc.tenq.services;
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.media.MediaRecorder;
+import android.util.Log;
+import android.os.Handler;
+
+import static java.lang.Math.abs;
+import static java.lang.Math.exp;
+
+public class RecorderService {
+
+    private static double mEMA = 0.0;
+    static final private double EMA_FILTER = 0.6;
+
+    private MediaRecorder mRecorder;
+    private final Context applicationContext;
+    private static Thread runner;
+    private boolean isRecorderOn = true; // TODO: change to false when not debugging
```

Comment:
So currently the app is always recording? Maybe it's best to push it with "false", so when other people are working on the app it's disabled

Reply:
You are right, I'll edit it.